src/Raven.Studio/typescript/viewmodels/resources/widgets/storageWidget.ts

```diff
@@ -109,6 +117,49 @@ class storageWidget extends websocketBasedWidget<Raven.Server.Dashboard.Cluster.
         if (stats) {
             action(stats);
         }
+
+        this.scaleDriveSize();
+    }
+
+    openWidgetSettings(): void {
+        const openSettingsDialog = new storageWidgetSettings(this.scaleToSize());
+
+        app.showBootstrapDialog(openSettingsDialog)
+            .done((scaleResult: boolean) => {
+                this.scaleToSize(scaleResult);
+                this.controller.layout(); // this will trigger saving to local storage
```

Comment:
let's create dedicated method for that instead of doing hacks, ok?

Reply:
done